### ISSUE TITLE
Upgrade from 2.1 fails due to rejected master password

### ISSUE THREAD

**Issue by derStephan** [nuxsmin/sysPass#1601]
**sysPass Version**
old: 2.1.6
new: 3.1 (latest) or 3.0.5, I first tried upgrading to 3.1 but that did not work, that's why I did it again with 3.0 with the same result.

**Describe the bug**
Upgrade following 
https://syspass-doc.readthedocs.io/en/3.1/updating/v3.html
https://syspass-doc.readthedocs.io/en/3.1/updating/v3.1.html

**To Reproduce**
1. Backup
2. download files and vendor
3. upload files and vendor to empty directory
4. copy “config.xml”, “key.pem” & “pubkey.pem” from the old “…/config” directory to “…/app/config” directory
5. load page, enter security upgrade code from config.xml
6. perform upgrade, message: successful
7. login with any user, message: master password not saved or wrong

after that, I have to enter user and password again together with the master password. If I enter these data there is a message: Master password wrong. 

But I am sure that the master password is correct. My master password is 100characters long and I had problems with that in the past: https://github.com/nuxsmin/sysPass/issues/519#issuecomment-293565271

Can it be, that this is the same or a similar problem again?

**Expected behavior**
possibility to log in, master password is accepted. 

**Event log**
[syspass.log](https://github.com/nuxsmin/sysPass/files/5418129/syspass.log)

**Platform (please complete the following information):**
 - OS: Linux, hostet on Plesk, PHP 7.2
 - Browser: Firefox 81.0.2

**Comment by derStephan** [comment]
I fumbled around a little more. It seems like the master password is accepted, when I enter only the first 72 characters of it. However, after that I get the message 
> warning An exception occured
> Please contact to the administrator

which is quite funny as I am the administrator and there is no reason given for this exception. 

So I went back to 2.1.6 to change the master password there. This fails as well with another unnamed error. What can I do now?

**Comment by nuxsmin** [comment]
Hello,

I'm trying to guess out what could be causing this issue. 

"Please contact to the administrator" states to tell the administrator the issue and the he/she would take a look to the server logs for more information :wink: 

Regards

**Comment by derStephan** [comment]
Nevermind. In the meantime, I wrote a little Password Manager with very little complexity and no frills on my own. Everything is done in the browser using the built in crypto-API, no dependencies to any external library or framework. It lacks all those features of sysPass that I did not need. 

Have a look if you wish https://github.com/derStephan/PasswordManager

@nuxsmin Thank you anyway, I used sysPass for some time and was always happy with it. But upgrading was always my problem. And I want to thank you for the support you are giving here for free and with very short delays. 👍👍👍
